changelog/b11997.dd

```diff
@@ -0,0 +1,7 @@
+Check for a D compiler next to rdmd first
```

Comment:
From the heading list, it won't be clear that this applies to rdmd, so how about:

"rdmd now prefers using a compiler binary from its directory, if one exists"